### DISCUSSION TITLE
“Recent repositories” does not show actual recent repositories

### DISCUSSION THREAD

**Discussion by acli** [community/community#29323] [Other Feature Feedback, Questions, & Ideas]
“Recent repositories” on main feed is not showing the most recent repository that has been created. It’s not even there if I press More.

**Comment by Bajorinas** [comment]
#29323

**Reply by skaillet92** [reply]
https://github.com/github/docs/issues/19712#issue-1337413743

**Reply by sadmankiba** [reply]
How is the issue related?

**Reply by magnum6actual** [reply]
This is actually super annoying. My recent list is all things I haven't touched in months.
